### ISSUE TITLE
Fix ptyxis VTE issues

### ISSUE THREAD

**Issue by castrojo** [ublue-os/bluefin#1286]
https://gitlab.gnome.org/chergert/ptyxis/-/issues/147#note_2109063

> Bluefin does not have a patched VTE for the a11y support. Upstream Fedora 40 has our patches applied now and so we should probably direct Bluefin to just use the upstream vte291-gtk4 package when building the OS layers.
We don't have access to AdwButtonRow which means this must be using the main branch and not the stable gnome-46 branch. That is unexpected and we should get Bluefin using the actual stable branch.

> I think we'll have ptyxis Fedora 40 packages upstream which might make this all easier very shortly as they can just pull that in rather than maintaining a downstream (is that the right word here?) package.

**Comment by KyleGospo** [comment]
Checked VTE in both 40 and Rawhide, still not new enough. 

Rebuilt Ptyxis with our VTE and gnome-46 branch.

**Comment by chergert** [comment]
Weird, because the `vte-profile` I have installed on F40 has the right patch, but when I look at https://src.fedoraproject.org/rpms/vte291/tree/f40 I don't see it.

**Comment by trafotin** [comment]
@chergert Would it be better to switch to the Flatpak of Ptyxis for future use? Given there was another issue opened 2 days ago about this issue and the profile works on the Flatpak.

For reference: https://gitlab.gnome.org/chergert/ptyxis/-/issues/159

**Comment by KyleGospo** [comment]
> @chergert Would it be better to switch to the Flatpak of Ptyxis for future use? Given there was another issue opened 2 days ago about this issue and the profile works on the Flatpak.
> 
> For reference: https://gitlab.gnome.org/chergert/ptyxis/-/issues/159

No. We wouldn't have a good way to install that for existing users that wouldn't fail for lack of an Internet connection, and may of the integrations we use with different desktop environments do not support the flatpak. That would also leave users with the ability to uninstall their only terminal.

We are very interested in the official Fedora RPM for Ptyxis however.

**Comment by KyleGospo** [comment]
> @chergert Would it be better to switch to the Flatpak of Ptyxis for future use? Given there was another issue opened 2 days ago about this issue and the profile works on the Flatpak.
> 
> For reference: https://gitlab.gnome.org/chergert/ptyxis/-/issues/159

No. We wouldn't have a good way to install that for existing users that wouldn't fail for lack of an Internet connection, and may of the integrations we use with different desktop environments do not support the flatpak. That would also leave users with the ability to uninstall their only terminal.

We are very interested in the official Fedora RPM for Ptyxis however.

**Comment by KyleGospo** [comment]
> @chergert Would it be better to switch to the Flatpak of Ptyxis for future use? Given there was another issue opened 2 days ago about this issue and the profile works on the Flatpak.
> 
> For reference: https://gitlab.gnome.org/chergert/ptyxis/-/issues/159

No. We wouldn't have a good way to install that for existing users that wouldn't fail for lack of an Internet connection, and may of the integrations we use with different desktop environments do not support the flatpak. That would also leave users with the ability to uninstall their only terminal.

We are very interested in the official Fedora RPM for Ptyxis however.

**Comment by castrojo** [comment]
We're just waiting for ptyxis to land in Fedora 40 (the deps have landed), and once we do that we'll be on the distro version and won't need to package it ourselves then we'll be good to go.

Once it lands I'll PR this.

**Comment by castrojo** [comment]
This will happen naturally when we drop F39
